### PR TITLE
feat: add note that clarifies the process of using Gravatar profile

### LINKED ISSUE #28553: Clarify how Gravatar profile pictures work on GitHub and how to update them
### Code of Conduct

- [x] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

- "[Personalizing your profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile)"

#### Content plan

Add a short final paragraph to the intro section, "Changing your profile picture."

- Tell readers that if they have a Gravatar image associated with the email they use for GitHub, it will be shown as their GitHub profile picture by default (rather than an identicon).
- Explain that to change their GitHub profile picture drawn from Gravatar, they can either update their Gravatar account, or they can upload a new image to GitHub, which will override the Gravatar image.

#### Problem to solve

Some people see their Gravatar image on GitHub and don't know where the image is coming from or how to change it. In some cases, they may have forgotten that they have an image associated with their Gravatar account.

GitHub users have expressed some confusion in the discussion posts below:

- [Where is my profile picture taken from? · community · Discussion #32845](https://github.com/orgs/community/discussions/32845)
- [How can I use Gravatar as my profile picture? · community · Discussion #38110](https://github.com/orgs/community/discussions/38110)

#### Who this affects

People who have an image associated with their Gravatar account and are confused about why this image is showing up on GitHub, and don't know how to change it.

#### How Gravatar images work on GitHub

- If you have a Gravatar image, it will automatically show up as your GitHub profile picture. 
- To change a GitHub profile picture that's drawn from Gravatar, users can either update their Gravatar account to have a new image, _or_ they can upload a new image to GitHub, which will override the Gravatar image.

### PR DESCRIPTION
**Pull request by CBID2**
### Why:
This pull request adds a note on how Gravatar profile pictures work on GitHub and how to update them. This addition helps users gain a better understanding of this process. 
## Closes: 
Closes #28553 

### Check off the following:

- [X] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [X] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by isaacmbrown** [comment]
Hi @CBID2, thanks for the contribution! In addition to the change @cmwilson21 identified above, would you be able to move your note so that it is the final paragraph in the intro section of "Changing your profile picture"? (This section starts on line 30). Because there's already a note here, and we don't generally use multiple notes in a row, you could also change your text to be a regular paragraph.

Thank you! ✨

**Comment by CBID2** [comment]
Made the changes @cmwilson21 and @isaacmbrown! :)

File: content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ When you sign up for an account, {% data variables.product.product_name %} provi
 {% ifversion ghec %}* Gravatar profile pictures are not supported with {% data variables.product.prodname_emus %}.{% endif %}
 
 {% endnote %}
+If your Gravatar image is associated with the email you use for {% data variables.product.product_name %}, it will be shown as your {% data variables.product.product_name %} profile picture by default (rather than an identicon). To change your {% data variables.product.product_name %} profile picture to your Gravatar image, you can either update your Gravatar account or upload a new image to{% data variables.product.product_name %}, which will override the Gravatar image.
```
Comment (isaacmbrown) [inline content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile.md:44]: A few wording suggestions and adding some whitespace.

```suggestion

If you use Gravatar, and your Gravatar image is associated with the email you use for {% data variables.product.product_name %}, the image will be shown as your {% data variables.product.product_name %} profile picture by default (rather than an identicon). To change your {% data variables.product.product_name %} profile picture, you can either update your Gravatar account or upload a new image to {% data variables.product.product_name %}, which will override the Gravatar image.
```

File: content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile.md [hunk 2]
```diff
@@ -180,13 +181,13 @@ If you select the "Busy" option, when people @mention your username, assign you
...
+| Badge                                                           | Program                                                        | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```
Comment (isaacmbrown) [inline content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile.md:184]: @CBID2 Were these changes deliberate? We have a linter that checks for trailing spaces so I think it would be best to reset these changes.

**Review by isaacmbrown — COMMENTED** [review]
Thanks for the updates @CBID2! Left a few more suggestions

File: content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile.md (hunk 2 above)
Comment (CBID2) [reply content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile.md:185]: Hi @isaacmbrown! :) I use [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) to check for syntax errors, so I use their Quick fix suggestion to remove the trailing spaces.

Comment (isaacmbrown) [reply content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile.md]: Hey @CBID2, I could still see the added whitespace in the tables so I pushed up a fix to revert the tables to how they were before. If you check the "Files changed" tab now you'll just see the note you added in the diff, which is what we want.

If you're ever in a situation again where you need to revert some of the changes to a file, I used the following command.

```
git checkout --patch main -- content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile.md
```

This lets you interactively choose which of your changes to the file you want to keep, and which changes you want to reset to how they are on `main`.

**Review by isaacmbrown — APPROVED** [review]
Thanks @CBID2, this is good to go! 🚀

**Comment by CBID2** [comment]
> Thanks @CBID2, this is good to go! 🚀

Yay! :) Thanks @isaacmbrown! :) The tests passed so can you merge my PR now? The auto merge did not work for some reason.

**Comment by CBID2** [comment]
> > Thanks @CBID2, this is good to go! 🚀
> 
> Yay! :) Thanks @isaacmbrown! :) The tests passed so can you merge my PR now? The auto merge did not work for some reason.

@isaacmbrown, the graphql test failed, causing my PR to be removed from the queue list. How can I fix this problem?

**Comment by isaacmbrown** [comment]
Hi @CBID2, this will be a temporary internal issue with our workflows, so no need to do anything from your side. I'll investigate the issue; I believe we're already looking into it
